### PR TITLE
test

### PR DESCRIPTION
**Pull request by nachinius**

### CODE REVIEW

**Comment by nachinius** [comment]
test commnet from vs

File: README.markdown [hunk 1]
```diff
@@ -1,6 +1,6 @@
 A [Giter8][g8] template for fp in scala
 
-### test 
+### test: deploy quickly locally
```
Comment (ignaciopeixoto) [inline README.markdown:3]: why this change?

Comment (nachinius) [reply README.markdown:3]: because i need it a change to test this

File: README.markdown [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
     LOC="/Users/Ignacio/dev/23-g8-scala-fp-starter"
     mkdir -p ~/dev/test-my-g8
     cd ~/dev/test-my-g8; g8 "file://$LOC" --name=g8test --force; cd g8test; sbt
```
Comment (nachinius) [inline README.markdown:6]: and this
